roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -23,6 +23,7 @@
   register: install_ovs_sysconfig
   notify:
   - restart openvswitch
+  - restart openvswitch pause
```

Comment:
A handler can notify another handler. Maybe just have `restart openvswitch` `notify: restart openvswitch pause` so anyone down the line doesn't have to remember to notify both?